org.testeditor.web.dropwizard.xtext/src/main/java/org/testeditor/web/xtext/index/BuildCycleManager.xtend

```diff
@@ -43,6 +46,43 @@ class BuildCycleManager {
 	var String[] staticSearchPaths = null
 
 	var ChangedResources currentChanges
+	
+	def void startRebuild() {
+		lastIndexState = new IndexState
+		val buildRequest = createBuildRequest
```

Comment:
you should be able to combine this and the following line to
`val buildRequest = createBuildRequest.addChangesForFull`